src/settings/MappingProfiles/detailsSections/edit/MappingInstanceDetails.js

```diff
@@ -0,0 +1,1186 @@
+                  </Row>
+                )}
+              />
+            </RepeatableActionsField>
+          </Col>
+        </Row>
+        <Row left="xs">
+          <Col
+            data-test-child-instances
+            xs={12}
+          >
+            <RepeatableActionsField
+              wrapperFieldName={getFieldName(31)}
+              legend={<FormattedMessage id={`${translationPrefix}.instance.field.childInstances.legend`} />}
+            >
+              <RepeatableField
+                fields={childInstances}
+                addLabel={<FormattedMessage id={`${translationPrefix}.instance.field.childInstances.addLabel`} />}
+                onAdd={() => onAdd(childInstances, 'childInstances', 'order', 31, initialFields, setReferenceTables)}
+                onRemove={index => onRemove(index, childInstances, 'order', 31, setReferenceTables)}
+                renderField={(field, index) => (
+                  <Row left="xs">
+                    <Col xs={6}>
+                      <Field
+                        component={TextField}
+                        label={<FormattedMessage id={`${translationPrefix}.instance.childInstances.field.subInstanceId`} />}
+                        name={getSubfieldName(31, 0, index)}
+                      />
+                    </Col>
+                    <Col
+                      data-test-child-type-of-relation
+                      xs={6}
+                    >
+                      <AcceptedValuesField
+                        okapi={okapi}
+                        component={TextField}
+                        label={<FormattedMessage id={`${translationPrefix}.instance.childInstances.field.instnaceRelationshipTypeId`} />}
+                        name={getSubfieldName(31, 1, index)}
+                        optionValue="name"
+                        optionLabel="name"
+                        wrapperLabel={`${translationPrefix}.wrapper.acceptedValues`}
+                        wrapperSourceLink="/instance-relationship-types?limit=1000&query=cql.allRecords=1 sortby name"
+                        wrapperSourcePath="instanceRelationshipTypes"
+                      />
+                    </Col>
+                  </Row>
+                )}
+              />
+            </RepeatableActionsField>
+          </Col>
+        </Row>
+      </Accordion>
+    );
+  };
+  const renderRelatedInstances = () => (
+    <Accordion
+      id="related-instances"
+      label={<FormattedMessage id={`${translationPrefix}.item.relatedInstances.section`} />}
+      separator
+    >
+      <></>
+    </Accordion>
+  );
+
+  return (
+    <AccordionSet>
+      {renderAdminData()}
+      {renderTitleData()}
+      {renderIdentifier()}
+      {renderContributor()}
+      {renderDescriptiveData()}
+      {renderInstanceNotes()}
+      {renderElectronicAccess()}
+      {renderSubject()}
+      {renderClassification()}
+      {renderInstanceRelationship()}
+      {renderRelatedInstances()}
```

Comment:
I think it’s better to put each function in a separate file, it will help to make this file more readable

Reply:
I'm not sure this is a really good idea, because there are 8 FOLIO record types for mapping profiles (`<MappingInstanceDetails>`, `<MappingItemDetails>` and so on) and each of this components, in turn, has own set of accordions, what's more accordions can repeat, but have different content

Reply:
As for me, we can create separate folder for each FOLIO record type and then set of separate js file for each section

Reply:
Yeah, splitting the file would be definitely a good idea IMO from the single responsibility principle and maintainability.

Reply:
Ok, you guys convinced me

Reply:
Done